OpenTok/OpenTok.cs

```diff
@@ -608,19 +616,70 @@ public void SetBroadcastLayout(string broadcastId, BroadcastLayout layout)
                 {
                     throw new OpenTokArgumentException("Could not set the layout. Either an invalid JSON or an invalid layout options.");
                 }
+                else if (layout.ScreenShareType != null && layout.Type != BroadcastLayout.LayoutType.BestFit)
+                {
+                    throw new OpenTokArgumentException($"Could not set screenShareLayout, when screenShareType is set, layout.Type must be bestFit, was {layout.ScreenShareType}");
```

Comment:
Commas, how do they work?
```suggestion
                    throw new OpenTokArgumentException($"Could not set screenShareLayout, when screenShareType is set layout.Type must be bestFit, was {layout.ScreenShareType}");
```